client/components/layout/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from '@mui/material';
+import Link from 'next/link';
+import React from 'react';
+
+import Navigation from './Navigation';
+
+const Header: React.FC = () => {
+  return (
+    <Box
+      component="header"
+      className="flex align-baseline justify-between p-3"
+    >
+      <Link href="/" passHref>
+        <button className="text-lg bg-transparent border-none cursor-pointer">
+          PairUp
+        </button>
```

Comment:
yup - because it's meant to be unstyled (basically, a replacement for the logo)

Reply:
Ahh, gotcha.

Actually, only the logout button should really be a button... the rest should probably be anchor tags. If you wanted, you could probably make it dynamic on the nav links... if it has an `onClick`, use a button... otherwise, use an anchor. 😂

That said... low priority. Not really an important change at this point.

Reply:
i think that's how it currently is - if there's an href prop, it'll render an anchor tag.  the logout button is already just a button since it only has an onclick handler and no href prop
![image](https://user-images.githubusercontent.com/5304277/236666451-4d3a15e5-0803-4de4-ae34-0c15a8061309.png)